study-builder/studies/pancan/emails/consent-complete.es.html

```diff
@@ -0,0 +1,15 @@
+#set( $text = {
+    "subject": "Gracias por proporcionar su consentimiento",
+    "salutation_self": "Estimado participante:",
+    "salutation_parent": "Estimado(a) padre/madre:",
+    "thank_you": "Gracias por unirse a Count Me In y por proporcionar su consentimiento a nuestra investigación.",
+    "what_to_expect": "A continuación, le enviaremos por correo postal un kit con instrucciones sobre cómo proporcionar una muestra de saliva.",
+    "what_to_expect_sample_request": "En algunos casos, también podríamos ponernos en contacto con usted para proporcionar una muestra de sangre (un tubo o dos cucharaditas por muestra), que puede extraerse en el consultorio de su médico, su clínica local o un laboratorio cercano. Si esto es necesario, le daremos instrucciones detalladas sobre cómo hacerlo. Es posible que obtengamos copias de {{~#if proxy_firstName~}}los registros médicos de su hijo(a){{~else~}}sus registros médicos{{~/if~}} y, después de revisarlos, es posible que también solicitemos una porción de la muestra del tumor almacenada al hospital donde se encuentra. Una vez que hayamos analizado {{~#if proxy_firstName~}}las muestras obtenidas de su hijo(a){{~else~}}las muestras obtenidas{{~/if~}}, devolveremos las muestras del tumor que no se hayan utilizado al departamento de patológica que nos las haya enviado.",
+    "next_steps": "Si aún no ha proporcionado información sobre dónde ha recibido atención {{~#if proxy_firstName~}}su hijo(a){{~else~}}usted{{~/if~}} para su cáncer, puede completar el proceso aquí:",
+    "button_title": "MEDICAL_RELEASE_FORM_BUTTON_TITLE",
```

Comment:
This is a tricky spot. The original English content from CMI did not have the correct text, and the incorrect text was translated into Spanish. Instead of incorporating a subtly incorrect title, I tried to make this bit and bold so we don't miss it.

Reply:
The roundtrip of going back to CMI then to external translators and back is probably not worth it in my view. If we have a bundle of other "little strings" like these that either got mistranslated or missed entirely, then it might be worth it at that point. But in this particular case, I see the English is just `Medical Release`. This is a common piece of string used in the release activity (e.g. name/title/other places I think). So I think we can just piece this together ourselves.

Reply:
On Angular side for similar cases we have another trick/convention  -
adding a `[ES]` prefix before the original string (for not translated strings). 
like: 
` "button_title": "[ES]Medical Release"  (in es.json)`

Reply:
Good to know- I've added a ticket for this bug to Jira: https://broadinstitute.atlassian.net/browse/DDP-6594